README.md

```diff
@@ -352,6 +352,34 @@ resulting Data.FS content are reset. It is therefore possible to pre-apply
 changes that will be part of the next release, but if there is a change that is
 not yet merged upstream, the layer should not be updated until it is.
 
+### Layer Mechanics Diagram
+
+This diagram illustrates how ZODBSync handles layering of file system source trees and the main rules of object composition:
+
+```mermaid
+flowchart TB
+  subgraph Layers
+    direction TB
+    A["Base Layer (Bottom-most)<br/>workdir: system base<br/>source: package objects"]
```

Comment:
That's not what the distinction between `workdir` and `source` is.

Also, there is no real difference between a "base layer" and a "middle layer". There are some things that are currently "special" about the top-most layer (which I actually tried to no longer name "custom layer", although that naming is probably still in some places in some comments and also once in the README), but no real difference any of the lower layers.